web_autoflip_demo/web_worker_pm/main.ts

```diff
@@ -0,0 +1,37 @@
+// global array buffer object to save data and pass to web worker
+declare let buffer: ArrayBuffer;
+
+// this function is called when the input loads a video
```

Comment:
According to the google style guide

- Use /** JSDoc */ comments for documentation, i.e. comments a user of the code should read.
- Use // line comments for implementation comments, i.e. comments that only concern the implementation of the code itself."

Replace the comment with /** */

Reply:
Also the style guide dictates "Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied "This method ..." before it." So replace the comments to begin with a verb believing there is an implicit "This method..."

Reply:
Done.